tests/Oryx.BuildImage.Tests/Python/CondaTests.cs

```diff
@@ -89,7 +89,7 @@ public void CanBuildAppWhichHasCondaEnvironmentYmlFile()
                 result.GetDebugInfo());
         }
 
-        [Fact, Trait("category", "vso-focal")]
+        [Fact, Trait("category", "latest")]
```

Comment:
Is these a reason we change the test image from vso focal to latest build image?

Reply:
@qianz2 Yes! This is just because VSO-focal doesn't actually have python 2 SDKs associated with it. We only have python 2 for stretch images, like latest